stats.go

```diff
@@ -52,11 +51,9 @@ const (
 	opencensusTaskKey         = "opencensus_task"
 	opencensusTaskDescription = "Opencensus task identifier"
 	defaultDisplayNamePrefix  = "OpenCensus"
-	version                   = "0.10.0"
+	version                   = "0.12.3"
```

Comment:
Looks like this hadn't been updated for a while. Updating this to prepare for the next release I will do after merging this PR

Reply:
If we do want the external code to be able to use this version in their custom user agent string, we should make this field public.

Also, this seems like a weird place to define version since it's not specific to `stats` AFAICT.
If you prefer to leave this code as is, that's fine by me.

Reply:
Yea I don't think I will move it. There seems to be quite a lot of global stuff defined in `stats.go`.

Also as you previously mentioned, it's unlikely external code would want to use this version no. in their custom user agent string, so I will leave this as a private field for now.